tedana/reporting/static_figures.py

```diff
@@ -463,3 +463,78 @@ def pca_results(criteria, n_components, all_varex, io_generator):
     pca_variance_explained_name = os.path.join(io_generator.out_dir, "figures", plot_name)
     plt.savefig(pca_variance_explained_name)
     plt.close()
+
+
+def plot_t2star_and_s0(
+    *,
+    io_generator: io.OutputGenerator,
+    mask: np.ndarray,
+) -> None:
+    """Create T2* and S0 maps and histograms.
+
+    Parameters
+    ----------
+    io_generator : :obj:`~tedana.io.OutputGenerator`
+        The output generator for this workflow
+    mask : (S,) :obj:`numpy.ndarray`
+        Binary mask used to apply to the data.
+    """
+    t2star_img = io_generator.get_name("t2star img")
+    s0_img = io_generator.get_name("s0 img")
+    mask_img = io.new_nii_like(io_generator.reference_img, mask.astype(int))
+    assert os.path.isfile(t2star_img), f"File {t2star_img} does not exist"
+    assert os.path.isfile(s0_img), f"File {s0_img} does not exist"
+
+    # Plot histograms
+    t2star_data = masking.apply_mask(t2star_img, mask_img)
+    t2s_p02, t2s_p98 = np.percentile(t2star_data, [2, 98])
+    t2star_histogram = f"{io_generator.prefix}t2star_histogram.svg"
+
+    fig, ax = plt.subplots(figsize=(10, 6))
+    ax.hist(t2star_data[t2star_data <= t2s_p98], bins=100)
+    ax.set_xlim(0, t2s_p98)
+    ax.set_title("T2*", fontsize=20)
+    ax.set_ylabel("Count", fontsize=16)
+    ax.set_xlabel("Seconds\n(limited to 98th percentile)", fontsize=16)
+    fig.tight_layout()
+    fig.savefig(os.path.join(io_generator.out_dir, "figures", t2star_histogram))
+
+    s0_data = masking.apply_mask(s0_img, mask_img)
+    s0_p02, s0_p98 = np.percentile(s0_data, [2, 98])
+    s0_histogram = f"{io_generator.prefix}s0_histogram.svg"
+
+    fig, ax = plt.subplots(figsize=(10, 6))
+    ax.hist(s0_data[s0_data <= s0_p98], bins=100)
+    ax.set_xlim(0, s0_p98)
+    ax.set_title("S0", fontsize=20)
+    ax.set_ylabel("Count", fontsize=16)
+    ax.set_xlabel("Arbitrary Units\n(limited to 98th percentile)", fontsize=16)
+    fig.tight_layout()
+    fig.savefig(os.path.join(io_generator.out_dir, "figures", s0_histogram))
+
+    # Plot T2* and S0 maps
+    t2star_plot = f"{io_generator.prefix}t2star_brain.svg"
+    plotting.plot_stat_map(
+        t2star_img,
+        bg_img=None,
+        display_mode="mosaic",
+        symmetric_cbar=False,
+        cmap="binary",
```

Comment:
```suggestion
        cmap="gray",
```
The binary color map is low-to-high white-to-black while the gray colormap is black-to-white, which is both how I think these images are typically viewed and how the text in the documentation describes the images.

Reply:
I'm concerned that a black-to-white colormap on a white background will result in high values disappearing, but I'm happy to try it out.

Reply:
![image](https://github.com/ME-ICA/tedana/assets/8228902/b1fd0f6c-6548-4f87-ad64-44f62a3243ea)

I think the black-to-white colormap makes the difference between uncovered voxels and high-T2* values unclear.

Reply:
In the two calls to  ` plotting.plot_stat_map` you could add `black_bg=True` to make a black background and then the lowest values will fade into black.
![image](https://github.com/ME-ICA/tedana/assets/7406227/bef5f76f-0870-4de6-86a0-7769d4b86231)